README.md

```diff
@@ -80,7 +80,7 @@ var myGeoJson = L.geoJSON(geoJsonData, { arrowheads: { <Options> } })
 
 Arrowheads offers a variety of options for rendering and styling arrowheads.  See the options table below.<br>
 <br>
-Arrowheads inherit all options from [L.Path](https://leafletjs.com/reference-1.6.0.html#path).  Arrowheads also inherit all options from their parent polylines, except `fill`, `fillOpacity`, and `smoothFactor`.  These can be changed manually when defining the arrowheads' options, but changing smoothFactor will result in improperly rendered arrows.<br>
```

Comment:
Not sure why we're removing `fill` from this list.  If you do `L.polyline({ fill: true }).arrowheads()`, the arrowhead will not be filled.  You'd have to do `L.polyline({ fill: true }).arrowheads({ fill: true })` to get filled arrowheads on a filled polyline.  That's why I say these properties are not inherited by the arrowhead.

Reply:
Understood! My mistake.